server/web/app.py

```diff
@@ -1,6 +1,6 @@
 from factory import create_app
 
-app = create_app()
+app = create_app('default')
```

Comment:
Make this the following
```python
app = create_app(os.getenv('CONFIG_LVL') or 'default')
```
So we can change the config level via environmental variables. This will be handy as we have `docker-compose.yml` for dev and `docker-compose.production.yml` for a production similar environment.